unit_tests/test_dockeropts.py

```diff
@@ -0,0 +1,38 @@
+from lib.dockeropts import DockerOpts
+
+
+class TestDockerOpts:
+
```

Comment:
Tests look good, to me!  What would d.add('','') do?  or d.add(None,None).  Would those work?